comms/Account.cpp

```diff
@@ -8,6 +8,56 @@
 #define ESP_EXCEPTION_LOG3(T) CReportingGSoapEspException<ns1__ArrayOfEspException> exceptions(T, __FILE__, __LINE__)
 
 #if _COMMS_VER >= 68200
+
+bool GetExceptionResponse(ns1__ArrayOfEspException *exceptions, CString &retMsg)
+{
+    bool noExceptions = true;
+    if (exceptions)
+    {
+        for (unsigned int i = 0; i < exceptions->Exception.size(); ++i)
+        {
+            if (exceptions->Exception[i]->Message)
+            {
+                if (!retMsg.IsEmpty())
+                    retMsg += _T("\r\n");
+
+                retMsg += exceptions->Exception[i]->Message->c_str();
+                noExceptions = false;
+            }
+        }
+    }
+    return noExceptions;
```

Comment:
`GetExceptionResponse` should return true IF it got an exception response and false otherwise.

Reply:
@GordonSmith agreed